src/Microsoft.AspNetCore.Mvc.TagHelpers/CacheTagHelper.cs

```diff
@@ -132,17 +132,38 @@ public override int Order
         public TimeSpan? ExpiresSliding { get; set; }
 
         /// <summary>
-        /// Gets or sets the <see cref="CacheItemPriority"/> policy for the cache entry.
-        /// </summary>
-        [HtmlAttributeName(CachePriorityAttributeName)]
-        public CacheItemPriority? Priority { get; set; }
```

Comment:
It doesn't have to. It would set the property on the context which would throw if you're using a feature the backing cache doesn't support.